.eslintrc.js

```diff
@@ -59,5 +59,11 @@ module.exports = {
       },
       parserOptions: { tsconfigRootDir: __dirname, project: ['./tsconfig.json'] },
     },
+    {
+      files: ['docs/src/pages/**/*.js', 'docs/src/pages/**/*.tsx'],
+      rules: {
+        'filenames/match-exported': ['error', [null, 'kebab', 'snake']],
```

Comment:
Keeping `kebab` and `snake` makes it to still suggest other names, although they can't be used because of another rule:

![image](https://user-images.githubusercontent.com/42154031/150373470-dec71fb0-de59-49e6-9252-2a84dd2ecce0.png)

Maybe?

```suggestion
      files: ['docs/src/pages/components/**/*.js', 'docs/src/pages/components/**/*.tsx'],
      rules: {
        'filenames/match-exported': ['error', [null]],
```